app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -167,14 +156,22 @@ class EventDetailsFragment : Fragment() {
                 rootView.feedbackRv.isVisible = true
                 rootView.noFeedBackTv.isVisible = false
             }
-            rootView.feedbackContainer.isVisible = it.isNotEmpty()
         })
 
         rootView.feedbackBtn.setOnClickListener {
             checkForAuthentication()
         }
 
-        eventViewModel.loadEvent(safeArgs.eventId)
+        val socialLinkLinearLayoutManager = LinearLayoutManager(context)
+        socialLinkLinearLayoutManager.orientation = LinearLayoutManager.HORIZONTAL
+        rootView.socialLinksRecycler.layoutManager = socialLinkLinearLayoutManager
+        rootView.socialLinksRecycler.adapter = socialLinkAdapter
+
+        eventViewModel.socialLinks.observe(viewLifecycleOwner, Observer {
+            socialLinkAdapter.addAll(it)
+            rootView.socialLinkContainer.visibility = if (it.isEmpty()) View.GONE else View.VISIBLE
```

Comment:
Can be written as `rootView.socialLinkContainer.isVisible = !it.isEmpty()`

Reply:
I'll check it again because I thought isVisible = false equals View.INVISIBLE, not View.GONE

Reply:
I should View.GONE. Check it. :+1:

Reply:
@anhanh11001 what about it?